### PR TITLE
Disallow running WiFi and ESPNow at the same time.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/resources/wifi_espnow_esp32.cc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+#include "../top.h"
+
+#ifdef TOIT_ESP32
+
+#include "wifi_espnow_esp32.h"
+#include "../resource_pool.h"
+
+namespace toit {
+#if defined(CONFIG_TOIT_ENABLE_WIFI) || defined(CONFIG_TOIT_ENABLE_ESPNOW)
```
Comment (kasperl) [inline src/resources/wifi_espnow_esp32.cc:24]: I think I'd prefer to compile out the whole file if the configs are not enabled, so maybe move this earlier?

File: src/resources/wifi_espnow_esp32.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// The license can be found in the file `LICENSE` in the top level
+// directory of this repository.
+
+#include "../resource_pool.h"
+namespace toit {
```
Comment (kasperl) [inline src/resources/wifi_espnow_esp32.h:17]: Add newline?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/resources/wifi_espnow_esp32.h (hunk 2 above)
Comment (floitsch) [reply src/resources/wifi_espnow_esp32.h:18]: done.

File: src/resources/wifi_espnow_esp32.cc (hunk 1 above)
Comment (floitsch) [reply src/resources/wifi_espnow_esp32.cc]: done.

**Pull request merged**